docs/DeveloperGuide.md

```diff
@@ -200,6 +200,17 @@ The HelpWindow dialog box provides users with the link to our User Guide and the
 
 ### Picture Command <a name="picture_command"></a>
 
+The Picture command allows users to add a profile picture to their contacts. The user can enter the command `picture INDEX`
+which would open a file browser to let the user choose any .jpg or .png file to be used as the profile picture, provided 
+`INDEX` is a valid index
+
+Represented below is the sequence diagram when `picture INDEX` is executed. For illustration purposes, `INDEX` refers to any valid index
+starting from 1 to the number of contacts in the displayed list.
```

Comment:
A very small nit here: is there any reason to break the sentence up into multiple lines? Should we just put them into one line and let markdown render them accrodingly?